### PR TITLE
Return detail info to edge core when registration node failed

### LINKED ISSUE #1827: Detail info of Edgecore node registration failed should be return to edged side
**What happened**:
When register node on the cloudhub, if failed, edged does not received any details info, the content is <nil>, should tend to cloudcore to get the error details. That error response should be returned to edged.

```
I0617 04:52:47.031896  164777 edged_status.go:392] Attempting to register node Khadas
I0617 04:52:47.117802  164777 process.go:164] received msg from cloud-hub:{Header:{ID:dfc332a4-5551-4105-88ea-2322048dc2b8 ParentID:d61f4ebd-e313-4338-b5fa-ca138839cc32 Timestamp:1592369567075 ResourceVersion: Sync:false} Router:{Source:edgecontroller Group:resource Operation:response Resource:default/node/Khadas} Content:}
E0617 04:52:47.117924  164777 edged_status.go:398] register node failed, error: <nil>
```

And from Cloud core log:
```
I0617 04:47:24.062700 2676445 upstream.go:174] message: c51b8b64-050c-47ab-b17f-c886d8a11451, resource type is: nodestatus
I0617 04:47:24.062707 2676445 upstream.go:176] message: c51b8b64-050c-47ab-b17f-c886d8a11451, operation type is: insert
I0617 04:47:24.062724 2676445 upstream.go:333] message: c51b8b64-050c-47ab-b17f-c886d8a11451, operation is: insert, and resource is node/Khadas/default/nodestatus/Khadas
E0617 04:47:24.066339 2676445 upstream.go:383] create node Khadas error: Node "Khadas" is invalid: metadata.name: Invalid value: "Khadas": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*') , register node failed
```

**What you expected to happen**:
Error detail should be returned to edged.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeEdge version(e.g. `cloudcore/edgecore --version`):  
1.3.1
**CloudSide Environment**:
- Hardware configuration (e.g. `lscpu`):
X86_64
- OS (e.g. `cat /etc/os-release`):
ubuntu focal
- Kernel (e.g. `uname -a`):
- Go version (e.g. `go version`): 
- Others:

**EdgeSide Environment**:
- edgecore version (e.g. `edgecore --version`):
Arm64
- Hardware configuration (e.g. `lscpu`):
- OS (e.g. `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Go version (e.g. `go version`): 
- Others:

### PR DESCRIPTION
**Pull request by kevin-zhaoshuai**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1827

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/edged/edged_status.go [hunk 1]
```diff
@@ -394,6 +394,9 @@ func (e *edged) registerNode() error {
 	res, err := beehiveContext.SendSync(edgehub.ModuleNameEdgeHub, *nodeInfoMsg, syncMsgRespTimeout)
 	if err != nil || res.Content != "OK" {
 		klog.Errorf("register node failed, error: %v", err)
+		if res.Content != "OK" {
+			klog.Errorf("reason from cloud core: %s", res.Content)
```
Comment (daixiang0) [inline edge/pkg/edged/edged_status.go:398]: s/reason/response?

**Comment by fisherxu** [comment]
@kevin-zhaoshuai  Has any updates?

**Comment by kevin-zhaoshuai** [comment]
@fisherxu  Sorry for the late response. Modified the code.

**Comment by fisherxu** [comment]
> @fisherxu Sorry for the late response. Modified the code.

Np, thanks for the fixing :)
/lgtm
/approve

**Pull request merged**